src/main/java/io/mosip/mimoto/service/impl/V2IssuersServiceImpl.java

```diff
@@ -160,7 +160,7 @@ private VCCredentialRequest generateVCCredentialRequest(IssuerDTO issuerDTO, Cre
                         .build())
                 .credential_definition(VCCredentialDefinition.builder()
                         .type(credentialsSupportedResponse.getCredential_definition().getType())
-                        .context(List.of("https://www.w3.org/2018/credentials/v1"))
+                        .context(List.of("https://challabeehyv.github.io/mimoto-config/insurance-context.json"))
```

Comment:
Why is this hardcoded?

Reply:
This is the new context file to support custom credential types. i will move this to properties

Reply:
But why is that important for inji web? The credentials are issued by Sunbird C and eSignet. Do we need this here?

Reply:
we have created two credential types HealthInsuranceCredential and LifeInsuranceCredential for demo purpose, e-signet credential issue request payload is expecting context to validate these credential types